developers_affiliations1.txt

```diff
@@ -14756,7 +14756,7 @@ Kludentwo: Kludentwo!users.noreply.github.com
 Kmannth: keith.mannthey!intel.com
 	Intel Corporation
 Kmoneal: Kmoneal!users.noreply.github.com, kenan.oneal!volunteers.acasi.info, kmacjoneal!gmail.com
-	Aspen Mesh
+	F5 Networks Inc.
```

Comment:
Question:
- was this affiliation bad and that user was never affiliated to `Aspen Mesh`? In this case the change is OK.
* or *
- is this a job change? In that case the change should be like:
```
   Aspen Mesh until YYYY-MM-DD
  F5 Networks Inc. from YYYY-MM-DD
```
?